src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] statistics = readStatistic(fromFileName);
+        writeStatistic(toFileName, statistics[0], statistics[1]);
+    }
```

Comment:
Use this template 
![image](https://github.com/mate-academy/jv-work-with-file/assets/63095184/b063e45b-4a31-4235-974f-55bc79b9f162)